### PR TITLE
Share extension initialization

### PR DESCRIPTION
**Pull request by jleandroperez**
#### Details:
In this PR we're exposing the Extension Initialization routine, so that it can be called from (anywhere else). This is meant to solve the "Extension doesn't get initialized after upgrading the app" glitch we've observed during our IPA tests.

Needs review: @astralbodies

### CODE REVIEW

File: WordPress/Classes/Services/AccountService.h [hunk 1]
```diff
@@ -86,6 +86,12 @@ extern NSString *const WPAccountEmailAndDefaultBlogUpdatedNotification;
 - (void)updateUserDetailsForAccount:(WPAccount *)account success:(void (^)())success failure:(void (^)(NSError *error))failure;
 
 /**
+ Initializes the WordPress iOS Extensions with the WordPress.com Default Account.
+ */
+- (void)setupExtensionsWithDefaultAccount;
```
Comment (astralbodies) [inline WordPress/Classes/Services/AccountService.h]: can we call this `setupAppExtensionsWithDefaultAccount` so it's super clear it's not class/Swift extensions ... I know I know...

Comment (jleandroperez) [inline WordPress/Classes/Services/AccountService.h]: Done!

**Comment by astralbodies** [comment]
:shipit:

**Comment by jleandroperez** [comment]
Thank you!

**Pull request merged**